things/api.py

```diff
@@ -476,30 +485,46 @@ def today(**kwargs):
     """
     Read Today's tasks into dicts.
 
-    Note: This might not produce desired results if the Things app hasn't
-    been opened yet today and the yellow "OK" button clicked for new tasks.
-    In general, you can assume that whatever state the Things app was in
-    when you last opened it, that's the state reflected here by the API.
+    Note: This method is a bit tricky. The API reflects the status of the database
+    when you last opened it with the app. However, there is the concept of "yellow"
+    time-related tasks. Some cases (e.g., scheduled or due tasks) are handled by the
+    API and are automatically shown. In other cases, however, such as with repeating
+    tasks, the database gets modified when opened with the app. These cases are not
+    covered before the app got launched.
```

Comment:
I find this hard to read, especially for someone who's never seen this code before. :-/

Here's an attempt:
1. The **Things Database** reflects the status of the Things app when it was last opened.
2. When you open the Things app, new to-dos might have appeared in the Today view. Those to-dos are Indicated by a yellow dot in the app.
3. We here try to predict what those new to-dos would be and return them as well.

Note to @AlexanderWillner: For readability, I'd avoid the technical term "task" if possible. Use "to-do" or "to-do or project" instead.